go/vt/vttablet/tabletmanager/vreplication/vreplicator_test.go

```diff
@@ -207,6 +207,7 @@ func TestDeferSecondaryKeys(t *testing.T) {
 	id := int32(1)
 	vsclient := newTabletConnector(tablet)
 	stats := binlogplayer.NewStats()
+	stats.Stop()
```

Comment:
Should this and the other cases here be a `defer`?

Reply:
I'd think so? The stats object mostly works even after stopping, but I think using `defer` here is going to more closely mimic the actual use of this object and is following the "principle of least suprise"?

Reply:
> Should this and the other cases here be a defer?
Yes, Fixed.